embedding-stores/chroma/deployment/src/main/java/io/quarkiverse/langchain4j/chroma/deployment/ChromaProcessor.java

```diff
@@ -43,7 +43,7 @@ public void createBean(
                         ParameterizedType.create(EmbeddingStore.class, ClassType.create(TextSegment.class)))
                 .defaultBean()
                 .setRuntimeInit()
-                .defaultBean()
+                .unremovable()
```

Comment:
Is this really correct?

Reply:
Now that the Dev UI page doesn't have a injection point for it anymore, I think we need to make it unremovable if we want to make it work for dev mode purposes. One test that uses Chroma was failing without this change

Reply:
At the very least, I would assume that we need the same thing everywhere

Reply:
Yeah right, I'll add it to others.. I think it's useful to have the embedding store working for Dev UI purposes even if the app doesn't have any injection point for it.

Reply:
Agreed